lib/between_meals/repo/hg.rb

```diff
@@ -120,14 +120,14 @@ def _username
 
       def check_refs(start_ref, end_ref)
         s = Mixlib::ShellOut.new(
-              "#{@bin} log -r #{start_ref}",
-              :cwd => @repo_path
+          "#{@bin} log -r #{start_ref}",
+          :cwd => @repo_path
             ).run_command
```

Comment:
the above to lines changing were correct, but now this one is no longre correct. It needs to come back... 4 spaces.